sunpy/net/jsoc/jsoc.py

```diff
@@ -500,7 +500,7 @@ def fetch(self, jsoc_response, path=None, progress=True, overwrite=False,
                                 wait=wait)
```

Comment:
you need to pass `**kwargs` through here (and in the call signature to `fetch`).